components/04-views/login/_login.scss

```diff
@@ -0,0 +1,167 @@
+$login__margin-bottom                       : $spacer--extra-large !default;
+$login__padding-top                         : $spacer--medium !default;
+$login__actions-margin                      : 0 0 $spacer--medium !default;
+
+$login__horizontal-padding                  : 38px !default;
+$login__button-margin                       : 0 $spacer--large 0 0 !default;
+$login__button-min-width                    : 160px !default;
+
+$login__tab-content-padding                 : $spacer--large 0 !default;
+$login__tab-title-background-active         : $gray-lightest !default;
+$login__tab-title-border                    : 8px solid $gray-lightest !default;
+$login__tab-title-horizontal-padding        : $spacer !default;
+$login__tab-title-horizontal-padding\@medium: $spacer--medium !default;
+
+$login__tabs-margin                         : 0 0 $spacer--extra-large 0 !default;
+$login__tabs-content-max-width\@medium      : 496px !default;
+$login__tabs-content-max-width\@large       : none !default;
+$login__tabs-content-margin                 : 0 auto !default;
+
+$login__forgot-password-margin              : 0 !default;
+$login__forgot-password-color               : $blue !default;
+$login__forgot-password-text-decoration     : none !default;
+
+$login__input-margin                        : 0 0 $spacer--medium !default;
+
+$login__sidebar-margin                      : 0 0 $spacer !default;
+
+$login__info-margin                         : 0 0 $spacer !default;
+$login__info-margin\@medium                 : 0 0 24px !default;
+
+$login__title-margin-bottom                 : $spacer--medium !default;
+$login__title-text-transform                : uppercase !default;
+
+$login__validation-tips-color               : $red !default;
+$login__validation-tips-margin              : 0 0 $spacer !default;
+
+.login {
+    padding-top: $login__padding-top;
+    margin-bottom: $login__margin-bottom;
+    &__title {
+        text-transform: $login__title-text-transform;
+        margin-bottom: $login__title-margin-bottom;
+    }
+    &__info {
+        margin: $login__info-margin;
+        text-align: center;
+        @include mq($screen-m) {
+            margin: $login__info-margin\@medium;
+            text-align: left;
+        }
+    }
+    &__content {
+        @include mq($screen-m) {
+            display: flex;
+            justify-content: space-between;
+            align-items: flex-start;
+        }
+    }
+    &__fieldset {
+        @include mq ($screen-m) {
+            max-width: 488px;
+        }
+    }
+    &__tabs {
+        margin: $login__tabs-margin;
+        width: 100%;
+        @include mq($screen-m) {
+            width: calc(50% - 8px);
```

Comment:
maybe we could use some variable for that ?

Reply:
done